src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -882,6 +882,45 @@ public void MultithreadedAnalyzeCommandBase_TargetFileSizeTestCases()
             }
         }
 
+        [Fact]
+        public void MultithreadedAnalyzeCommandBase_ErrorWhenHashing()
```

Comment:
>MultithreadedAnalyzeCommandBase_ErrorWhenHashing [](http://example.com/codeflow?start=20&length=48)

this is to add a test for previous pr about `NullReferenceException`.  
without that fix, this test will fail with  `NullReferenceException`